production/helm/loki/values.yaml

```diff
@@ -285,6 +285,13 @@ loki:
       s3ForcePathStyle: false
       insecure: false
       http_config: {}
+      backoff_config:
```

Comment:
This should be the empty object so existing customers don't get different behaviour when upgrading.

Reply:
This isn't empty yet. Are you still working on this?

Reply:
whoops! didn't pull after committing your suggestion on the version so missed that it didn't get pushed up. this should be empty now and the docs updated to reflect it